python/ray/rllib/models/multiagentfcnet.py

```diff
@@ -37,10 +37,24 @@ def _init(self, inputs, num_outputs, options):
 
         # check for a shared model
         shared_model = custom_options.get("multiagent_shared_model", 0)
+        shared_model = shared_model or isinstance(shared_model, list) # for heterogeneous shared models
+        # the list contains how many agents share each model
+        shared_model_list = custom_options.get("multiagent_shared_model", [len(hiddens)])
+
         reuse = tf.AUTO_REUSE if shared_model else False
         outputs = []
+        # keeps track of how many models we have set shared so far
```

Comment:
"set shared" --> "set as shared"?

Reply:
Yes, this is the intent of the request. I'll modify that